CMakeLists.txt

```diff
@@ -48,9 +48,9 @@ macro(cpr_option OPTION_NAME OPTION_TEXT OPTION_DEFAULT)
     message(STATUS "  ${OPTION_NAME}: ${${OPTION_NAME}}")
 endmacro()
 
-option(BUILD_SHARED_LIBS "Build libraries as shared libraries" ON)
```

Comment:
This line never should have existed in the first place. Thanks for noticing this.

Reply:
After a short discussion with @KingKili it turns out, I was wrong with this line. It is an `option`, which will set `BUILD_SHARED_LIBS` only and if only in case `BUILD_SHARED_LIBS` is not defined already. In this case, if nothing is set at all for `BUILD_SHARED_LIBS`, we will set it to `ON`. So, it is fine, how it is right now. It is no `cpr_option` to reduce complexity.

Reply:
Sorry for wasting your time ...